src/v2/Apps/Artwork/Components/ArtworkVideoPlayer/ArtworkVideoPlayer.tsx

```diff
@@ -0,0 +1,87 @@
+import React, { FC } from "react"
+import { Box, ResponsiveBox } from "@artsy/palette"
+import { createFragmentContainer, graphql } from "react-relay"
+import { ArtworkVideoPlayer_artwork } from "v2/__generated__/ArtworkVideoPlayer_artwork.graphql"
+import styled from "styled-components"
+
+interface ArtworkVideoPlayerProps {
+  activeIndex: number
+  artwork: ArtworkVideoPlayer_artwork
+  small?: boolean
+}
+
+const ArtworkVideoPlayer: FC<ArtworkVideoPlayerProps> = ({
+  activeIndex,
+  artwork: { figures },
+  small,
+}) => {
+  if (!figures) {
+    return null
+  }
+
+  const activeVideo = figures[activeIndex]
+
+  if (!activeVideo || activeVideo.type === "%other") {
```

Comment:
Is `%other` correct or is the `%` a typo?

Reply:
its the union type placeholder. (not a typo, kinda annoying)

Reply:
Strange!